src/app.js

```diff
@@ -6,6 +6,24 @@ const mongoose = require("mongoose");
 
 //Routes
 const shopRoutes = require("./routes/shops");
+const userRoutes = require("./routes/user.js");
+
+
+// const jwt   = require('express-jwt');
```

Comment:
I understand your confusion here. I'll mark the PR as Requested changes just so we can work on this together :)